vm/src/interpreter.cpp

```diff
@@ -87,17 +98,18 @@ void run(constant_pool pool, int ip, const char *bytes, size_t size) {
             // Fork and exec the process
             pid_t pid = fork();
             if (pid == 0) {
-                // Execute the process
+                // Execute the process then exit
                 execvp(argv[0], argv);
             } else {
                 for (int i = 0; i < frame_size; i++) {
                     delete[] argv[i];
                 }
                 delete[] argv;
+                int status;
```

Comment:
```suggestion
                int status = 0;
```
Initializing values never hurt :p

Reply:
useless

Reply:
What is your guarantee that the function below will actually mutate this value? Having dangling values is a very important thing to avoid in memory unsafe languages like C or C++.

Reply:
mmh i see indeed, my bad !